storage.bs

```diff
@@ -178,7 +178,21 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+The <dfn export>application cache site storage usage</dfn> for an <a for=/>origin</a>
+<var>origin</var> is a rough estimate of the amount of bytes used in <a>Application Cache</a>
+in <var>origin</var>'s <a>site storage unit</a>. [[!HTML]]
```

Comment:
This one can contain opaque responses right? Given that this is a new place where we expose that information, I think we should also detail how to address the issue. Same for caches below.

Reply:
I intentionally pulled the security part out into a separate PR in which I address the opaque response + padding issues.  If you believe they should be merged in the PR, I can consolidate the two.  Thoughts?

Reply:
The other PR did not have the necessary wording either, right? But yeah, I think they should be included here and should probably be noted closer to where we define these features. Not in a separate section somewhere. The separate section seems like a fine place to highlight some of the trickier areas, but in general we should note security requirements right alongside general implementation requirements.

Reply:
I've taken a stab at adding a recommendation for padding.  Please take a look when you get a chance, thanks!